js/plugins/vertexai/src/embedder.ts

```diff
@@ -145,7 +153,8 @@ export function defineVertexAIEmbedder(
             task_type: options?.taskType,
             title: options?.title,
           };
-        })
+        }),
+        { outputDimensionality: options?.outputDimensionality }
```

Comment:
why is not along with other options (lines 152-154)?

Reply:
Thank you for your question.
Based on the API interface, it seems that `outputDimensionality` cannot be included under `instances`, and must be placed under `parameters`.
(documentation here: [Get text embeddings](https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings#get_text_embeddings_for_a_snippet_of_text))

After verifying the actual behavior, I found that when `outputDimensionality` is placed under `instances` (lines 152-154), it is ignored, and the default dimensionality is returned.

Reply:
Perfect! Just making sure.